core/src/main/java/org/jdbi/v3/Jdbi.java

```diff
@@ -289,8 +377,33 @@ public void registerColumnMapper(ColumnMapper<?> mapper) {
      *
      * @param factory the column mapper factory
      */
-    public void registerColumnMapper(ColumnMapperFactory factory) {
+    public Jdbi registerColumnMapper(ColumnMapperFactory factory) {
         config.mappingRegistry.addColumnMapper(factory);
+        return this;
+    }
+
+    /**
+     * Register a row mapper which will have its parameterized type inspected to determine what it maps to
+     *
+     * Will be used with {@link Query#mapTo(Class)} for registered mappings.
```

Comment:
maybe later, we should do a separate pass over the javadoc.  Trying to get the code cleaned up right now :)